Changes.md

```diff
@@ -1,3 +1,13 @@
+1.0.x.x (relative to 1.0.2.0)
+=======
+
+API
+---
+
+ViewportGadget : Added setPostProcessShader.  This allows the main layer to be rendered to a framebuffer, and processed by a shader before being displayed.  Useful for applying color transforms on the GPU after rendering.
+- GafferImageUI : Added OpenColorIOAlgo::displayTransformToFramebufferShader.  Converts an OCIO processor to a shader suitable for use with setPostProcessShader
+- ImageView : Apply color transform on viewport instead of ImageGadget.  Should not impact user visible behaviour, but paves the way for future work.
```

Comment:
We avoid the weird programmery imperative mood in the Changes file, since it is user facing.

We should mention the breaking changes to the ImageGadget API.